config/environments/test.rb

```diff
@@ -22,7 +22,8 @@
     env.cache = ActiveSupport::Cache.lookup_store(:memory_store)
   end
 
-  config.assets.compile = ENV['TEST_SUITE'] == 'spec:javascript'
+  config.assets.debug = true
+  config.assets.compile = %w(spec:javascript spec).include?(ENV['TEST_SUITE'])
```

Comment:
Will need UI review on keeping these two changes.  We need to compile assets so tests don't raise a warning on missing assets from the pipeline.

Reply:
cc @h-kataria @himdel

Reply:
I noticed that the disabling of live compilation in other test suites causes a failure like:
```
DEPRECATION WARNING: The asset "layout/brand.svg" is not present in the asset pipeline.Falling back to an asset that may be in the public folder.
This behavior is deprecated and will be removed.
To bypass the asset pipeline and preserve this behavior,
use the `skip_pipeline: true` option.
 (called from image_path at /home/travis/build/jrafanie/manageiq-api/app/controllers/api/api_controller.rb:89)
```

Seen in https://github.com/ManageIQ/manageiq-api/pull/606